experiment_prototype/scan_classes/sequences.py

```diff
@@ -3,43 +3,46 @@
 """
     sequences
     ~~~~~~~~~
-    This is the module containing the Sequence class. The Sequence class contains the 
-    ScanClassBase members, as well as a list of pulse dictionaries, 
-    the total_combined_pulses in the sequence, power_divider, last_pulse_len, ssdelay, 
-    seqtime, which together give sstime (scope synce time, or time for receiving, 
-    and numberofreceivesamples to sample during the receiving window (calculated using 
-    the receive sampling rate). 
+    This is the module containing the Sequence class. The Sequence class contains the
+    ScanClassBase members, as well as a list of pulse dictionaries,
+    the total_combined_pulses in the sequence, power_divider, last_pulse_len, ssdelay,
+    seqtime, which together give sstime (scope synce time, or time for receiving,
+    and numberofreceivesamples to sample during the receiving window (calculated using
+    the receive sampling rate).
 
     :copyright: 2018 SuperDARN Canada
     :author: Marci Detwiller
 """
 
 import math
-
+import numpy as np
+from scipy.constants import speed_of_light
+import copy
 from operator import itemgetter
+import collections
 
-from sample_building.sample_building import make_pulse_samples, calculate_first_rx_sample_time
+from sample_building.sample_building import get_samples
 from experiment_prototype.scan_classes.scan_class_base import ScanClassBase
-
+from experiment_prototype.experiment_exception import ExperimentException
 class Sequence(ScanClassBase):
-    """  
-    Set up the sequence class. 
-    
+    """
+    Set up the sequence class.
+
     **The members of the sequence are:**
-    
+
     pulses
-        a list of pre-combined, pre-sampled pulse dictionaries (one dictionary = one 
-        basic pulse of single frequency). The dictionary keys are: isarepeat, 
-        pulse_timing_us, slice_id, slice_pulse_index, pulse_len, 
-        intra_pulse_start_time, combined_pulse_index, pulse_shift, iscombined, 
+        a list of pre-combined, pre-sampled pulse dictionaries (one dictionary = one
```

Comment:
Can you update all these docs on class members? Looks like pulses dict no longer exists, and I noticed basic_slice_pulses is not listed here. There may be others removed and replaced